election-api-javascript/package.json

```diff
@@ -8,7 +8,8 @@
     "test": "jest"
   },
   "author": "BBC News and Weather Apps",
-  "license": "ISC",
+  "private": true,
```

Comment:
Don't think we need this line?

Reply:
This there to stop people accidentally publishing the project to NPM (because that's not really something we want to do with this project — it's not intended to be installed via a registry, it's intended to be checked-out from GitHub).